extensions/sql-migration/src/api/azure.ts

```diff
@@ -223,8 +235,8 @@ export interface MigrationController {
 }
 
 export interface GetMigrationControllerAuthKeysResult {
-	keyName1: string,
-	keyName2: string
+	authKey1: string,
```

Comment:
could you please add some comments why are there 2 auth keys in the result?

Reply:
Both the auth keys are for the same thing. There is no functional difference between the 2 authkeys. You can use either one of them.

Reply:
Is there documentation you could link to that discusses this? I'm assuming there's two for key rotation purposes.